client/snap_op.go

```diff
@@ -34,6 +35,19 @@ func (client *Client) InstallSnap(name, channel string) (string, error) {
 	return client.doAsync("POST", path, nil, body)
 }
 
+// SideloadSnap sideloads the snap with the given path, returning the UUID
+// of the background operation upon success.
+//
+// XXX: add support for "X-Allow-Unsigned"
+func (client *Client) SideloadSnap(path string) (string, error) {
```

Comment:
Can we please name this as `InstallSnapFille`?  We'll use that same operation to install official snaps properly signed and which will be indistinguishable from ones coming from the store after installation.